zoo-css-challenge/style.css

```
@@ -105,27 +107,41 @@ footer {
 
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
+  border: 5px solid #be4242;
 }
 
 .image-container img:first-of-type {
-  vertical-align: bottom;
+  border: 5px solid purple;
+}
+.image-container img:last-of-type {
+  border: 5px solid goldenrod;
 }
-
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
+.tigers h2,h3{
```

```suggestion
.tigers h2, .tigers h3{
```
The `h3` selector applies to all `h3` headings on the page. The task is to update it only for the tigers section so we need to update the selector to `.tigers h3`